docs/framework/data/adonet/ef/language-reference/linq-to-entities.md

```diff
@@ -84,6 +84,6 @@ LINQ to Entities provides Language-Integrated Query (LINQ) support that enables
   
 ## See also
 - [Known Issues and Considerations in LINQ to Entities](../../../../../../docs/framework/data/adonet/ef/language-reference/known-issues-and-considerations-in-linq-to-entities.md)
-- [LINQ (Language-Integrated Query)](https://msdn.microsoft.com/library/a73c4aec-5d15-4e98-b962-1274021ea93d)
+- [LINQ (Language-Integrated Query)](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2013/bb397926(v=vs.120))
```

Comment:
There are 13 topics that use this link:
https://github.com/dotnet/docs/search?q=a73c4aec-5d15-4e98-b962-1274021ea93d&unscoped_q=a73c4aec-5d15-4e98-b962-1274021ea93d

They should all be changed to something similar to
```suggestion
- [Language-Integrated Query (LINQ) - C#](../../../../../csharp/programming-guide/concepts/linq/index.md)
- [Language-Integrated Query (LINQ) - Visual Basic](../../../../../visual-basic/programming-guide/concepts/linq/index.md)
```